tests/datastore_elasticsearch/conftest.py

```diff
@@ -39,8 +40,20 @@
 ES_URL = f"http://{ES_SETTINGS['host']}:{ES_SETTINGS['port']}"
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="function")
```

Comment:
Maybe replace `ES_VERSION = tuple([int(n) for n in get_package_version("elasticsearch").split(".")])` with the following while we're in here?
```
ES_VERSION = get_package_version_tuple("elasticsearch")
```